src/adapter/scriptSkipper.ts

```diff
@@ -87,10 +91,15 @@ export class ScriptSkipper {
     return this._isUrlSkippedMap.get(url)!;
   }
 
-  public setAllNodeInternalsToSkip(nodeInternalsNames: string[]): void {
-    let fullLibNames = nodeInternalsNames.map(name => name + '.js');
-    fullLibNames.push('^internal/.+\.js|');
-    this._nodeInternalsRegex = this._createRegexString(fullLibNames);
+  public setAllNodeInternals(nodeInternalsNames: string[]): void {
+    this._allNodeInternals = nodeInternalsNames.map(name => name + '.js');
+  }
+
+  public toggleSkipFileStatus(url: string | undefined): void {
```

Comment:
Should this be calling `this._updateBlackboxedUrls` too, or how does that actually get called after toggling?

Reply:
Yes, it should! Sorry about that. Fixed